packages/react-core/src/components/OverflowMenu/OverflowMenuDropdownItem.tsx

```diff
@@ -14,12 +12,13 @@ export interface OverflowMenuDropdownItemProps extends React.HTMLProps<HTMLDivEl
 export const OverflowMenuDropdownItem: React.SFC<OverflowMenuDropdownItemProps> = ({
   children,
   isShared = false,
-  index
+  index,
+  ...additionalProps
 }: OverflowMenuDropdownItemProps) => (
   <OverflowMenuContext.Consumer>
     {value =>
       (!isShared || value.isBelowBreakpoint) && (
-        <DropdownItem component="button" index={index}>
+        <DropdownItem component="button" index={index} {...additionalProps}>
```

Comment:
I'd rather be explicit about what props are passed down Vs anything random. The issue is only asking for DropdownItem props to be passed thru. Can we destructure the valid props here?

Reply:
I don't think we can, dropdown items props extends `HTMLDivElement` and that one contains much much more props.
Unless, we do not need to pass `HTMLDivElement` props, I believe we can do that explicitly.

Reply:
Looking at other components, it appears props are passed thru without any validation.